e2e/ansible/playbooks/hyperconverged/test-k8s-storage-pool/test-storage-pool.yml

```diff
@@ -4,33 +4,50 @@
     - storage-pool-vars.yml
 
   tasks:
+  
+   - block:
+
    
-    #Deploying operator in default and pvc in percona namespace
-    - include: deploy-openebs-pvc.yml 
-        operator_namespace=default
-        namespace=percona
-
-    - include: cleanup.yml
-        operator_namespace=default
-        namespace=percona
+       #Deploying operator in default and pvc in percona namespace
+       - include: deploy-openebs-pvc.yml 
+           operator_namespace=default
+           namespace=percona
+
+       - include: cleanup.yml
+           operator_namespace=default
+           namespace=percona
     
-    #Deploying operator in openebs and pvc in default namespace
-    - include: deploy-openebs-pvc.yml
-        operator_namespace=openebs
-        namespace=default
-
-    - include: cleanup.yml
-        operator_namespace=openebs
-        namespace=default
-
-    #Deploying operator in openebs and pvc in openebs namespace
-    - include: deploy-openebs-pvc.yml
-        operator_namespace=openebs
-        namespace=percona
-
-    - include: cleanup.yml
-        operator_namespace=openebs
-        namespace=percona
-
-
+       #Deploying operator in openebs and pvc in default namespace
+       - include: deploy-openebs-pvc.yml
+           operator_namespace=openebs
+           namespace=default
+
+       - include: cleanup.yml
+           operator_namespace=openebs
+           namespace=default
+
+       #Deploying operator in openebs and pvc in openebs namespace
+       - include: deploy-openebs-pvc.yml
+           operator_namespace=openebs
+           namespace=percona
+      
+       - include: cleanup.yml
+           operator_namespace=openebs
+           namespace=percona
+
+
+       - set_fact:
+           flag: "Pass"
+
+     rescue: 
+       - set_fact: 
+           flag: "Fail"
+
+     always:
+
+       - name: Send slack notification
+         slack: 
```

Comment:
@Swarnalatha-K Can you delete the test artifacts in all the playbooks at the end of test.